AirCasting/Dashboard/Reordering/ReorderingDashboardViewModel.swift

```diff
@@ -10,16 +10,37 @@ class ReorderingDashboardViewModel: ObservableObject {
     
     @Published var currentlyDraggedSession: Sessionable?
     @Injected private var uiStorage: UIStorage
+    @Injected private var externalSessionsStore: ExternalSessionsStore
+    @Injected private var measurementStreamStorage: MeasurementStreamStorage
     
     init(sessions: [Sessionable], thresholds: [SensorThreshold]) {
         self.sessions = sessions
         self.thresholds = thresholds
     }
     
+    func clear(session: Sessionable) {
+        if session.isExternal {
+            externalSessionsStore.deleteSession(uuid: session.uuid) { result in
+                switch result {
+                case .success:
+                    Log.info("Deleted external session")
+                case .failure(let error):
+                    Log.error("Failed to delete External Session: \(error)")
```

Comment:
If `clear` fails we'll have an invalid UI state since the removal didn't succeed. It's quite unlikely, but maybe we can add a reload here? So that whenever this delete fails the session will pop back on the UI

Reply:
Hm, but then the user might try to swipe again and again and most likely it still wouldn't work. I'm not sure what is the best way to handle this failure, but I would just keep it as is and keep an eye out for this error popping up in firebase 👀